README.md

```diff
@@ -1,22 +1,27 @@
 # ambit
 
+[![Rust](https://github.com/plamorg/ambit/actions/workflows/rust.yml/badge.svg)](https://github.com/plamorg/ambit/actions/workflows/rust.yml)
```

Comment:
This should not be named `Rust` - try `CICD`, `CI`, or look at other projects for inspiration.

Reply:
Changing this in the markdown would only change the alternate text; it won't impact what it looks like when rendered... But I will change this anyway.